app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -156,56 +172,69 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
                                 checked={selected.some((i) => i.id === item.id)}
                               />
                             </Col>
-                            <Col className="tone-date">
-                              <Row>
-                                <Sentiment
-                                  value={item.tonePools?.length ? item.tonePools[0].value : 0}
-                                />
-                                <div className="date text-content">
-                                  {new Date(item.publishedOn).toDateString()}
-                                </div>
-                                <span className="divider"> | </span>
-                                <div className="source text-content">{item.source?.name}</div>
-                                <Show visible={!!item.series?.name}>
-                                  <span className="divider"> | </span>
-                                  <div className="series text-content">{item.series?.name}</div>
-                                </Show>
-                              </Row>
+                            <Col className="sentimentColumn">
+                              <Sentiment
+                                value={item.tonePools?.length ? item.tonePools[0].value : 0}
+                              />
+                            </Col>
+                            <Col className="dateColumn col-date">
+                              <div className="date">{formatDate(item.publishedOn)}</div>
                             </Col>
+                            <Col className="sourceColumn">
+                              {item.contentType === ContentTypeName.AudioVideo
+                                ? item.series?.name
+                                : item.source?.name}
+                            </Col>
+                            <Col className="headlineColumn">
+                              <div className="headline">{item.headline}</div>
+                            </Col>
+                            {(item.mediaType?.name === 'Daily Print' ||
+                              item.mediaType?.name === 'Weekly Print' ||
+                              item.mediaType?.name === 'Online') && (
+                              <Col className="linkColumn">{item.page}</Col>
+                            )}
+                            {(item.mediaType?.name === 'Events' ||
+                              item.mediaType?.name === 'News Radio' ||
+                              item.mediaType?.name === 'Talk Radio' ||
+                              item.mediaType?.name === 'TV / Video News') && (
+                              <Col className="linkColumn">
+                                <div
+                                  className="new-window"
+                                  onClick={(e) => {
+                                    e.stopPropagation();
+                                    window.open(`/view/${item.id}`, '_blank');
```

Comment:
Is the desired result a new tab showing the story?   If so this looks good to me.